TYPO3.Flow/Classes/TYPO3/Flow/Core/Booting/Scripts.php

```diff
@@ -159,7 +160,15 @@ public static function initializeConfiguration(Bootstrap $bootstrap)
         $settings = $configurationManager->getConfiguration(\TYPO3\Flow\Configuration\ConfigurationManager::CONFIGURATION_TYPE_SETTINGS, 'TYPO3.Flow');
 
         $environment = new \TYPO3\Flow\Utility\Environment($context);
-        $environment->setTemporaryDirectoryBase($settings['utility']['environment']['temporaryDirectoryBase']);
+        if (isset($settings['utility']['environment']['temporaryDirectoryBase'])) {
+            $defaultTemporaryDirectoryBase = FLOW_PATH_DATA . '/Temporary';
+            if (FLOW_PATH_TEMPORARY_BASE !== $defaultTemporaryDirectoryBase) {
+                throw new \TYPO3\Flow\Exception(sprintf('It seems like the PHP default temporary base path has been changed from "%s" to "%s" via the FLOW_PATH_TEMPORARY_BASE environment variable. If that variable is present the TYPO3.Flow.utility.environment.temporaryDirectoryBase setting must not be specified!', $defaultTemporaryDirectoryBase, FLOW_PATH_TEMPORARY_BASE), 1447707261);
```

Comment:
"If that variable is present" -> there's a comma missing after "present", which would help a bit understanding it at first read.